src/Tizen.NUI/src/public/Rendering/Renderer.cs

```diff
@@ -27,14 +27,19 @@ namespace Tizen.NUI
     /// <since_tizen> 3 </since_tizen>
     public class Renderer : Animatable
     {
+        private Geometry CurrentGeometry = null;
```

Comment:
You should dispose owned object when owner object was disposed. if owned object and owner are IDisposable

Reply:
I tested it is disposed automatically by DisposeQueue with DisposeTest.cs
could you check https://github.sec.samsung.net/dotnet/NUIBackend/issues/34#issuecomment-1218728 ?

Reply:
Thank you for reviewing. I added  `protected override void Dispose(bool disposing)`